packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -656,9 +656,11 @@ void WebView::OnNavigationPolicy(void* data, Evas_Object* obj,
 
 void WebView::OnEvaluateJavaScript(Evas_Object* obj, const char* result_value,
                                    void* user_data) {
-  FlMethodResult* result = static_cast<FlMethodResult*>(user_data);
-  result->Success(flutter::EncodableValue(result_value));
-  delete result;
+  if (result_value) {
+    FlMethodResult* result = static_cast<FlMethodResult*>(user_data);
+    result->Success(flutter::EncodableValue(result_value));
+    delete result;
+  }
```

Comment:
Obviously this will cause a leak. Should be something like this:

```cpp
  FlMethodResult* result = static_cast<FlMethodResult*>(user_data);
  if (result_value) {
    result->Success(flutter::EncodableValue(result_value));
  } else {
    result->Success();
  }
  delete result;
```